gaffer-as-a-service/pom.xml

```diff
@@ -30,15 +30,14 @@
 
     <modules>
         <module>gaas-common</module>
-        <module>gaffer-controller</module>
         <module>gaas-rest</module>
     </modules>
 
     <properties>
         <gaffer.version>1.17.1</gaffer.version>
         <kubernetes.version>14.0.0</kubernetes.version>
         <springboot.version>2.5.8</springboot.version>
-        <fasterxml.jackson.version>2.11.2</fasterxml.jackson.version>
+        <fasterxml.jackson.version>2.9.8</fasterxml.jackson.version>
```

Comment:
Jackson downgrade?

Reply:
Think that this must have been a mistake while merging. Have resolved